src/pages/playlist/Playlist.jsx

```diff
@@ -30,10 +28,11 @@ export function Playlist() {
             </h2>
           )}
           {!loading &&
+            playlists &&
             playlists.map(playlist => (
               <PlaylistCard playlist={playlist} key={playlist._id} />
             ))}
-          {!loading && playlists.length < 1 && (
+          {!loading && playlists && playlists.length < 1 && (
             <div className="add-videos">
```

Comment:
Any reason behind using the different formats of class names? some places its joined with  '__' , here it's '-' ?

Reply:
the double underscore( __ ) is from my component library, which I will change to single dash(-).